### DISCUSSION TITLE
App doesn't start in foreground anymore after upgrading target API from 32 to 33

### DISCUSSION THREAD

**Discussion by veitbjarsch** [HaylLtd/cordova-plugin-background-geolocation#158] [Q&A]
Since yesterday it is mandetory for new apps to use at least the android-targetSdkVersion 33. Before I compiled with version 33 but used 32 as "android-targetSdkVersion". Since I changed the tagret sdk version the foreground message doesn't appear anymore. I already try to debug the issue but I could not find any error indicating the problem. I just changed the target version. Once I revert it back it will work without a problem. 
Do I need to change something within the config if I upgrade to the new API version?

May @HarelM can help me here? As I can see you are already using the sdk version 33 in your project. Did you face any similar issue?

I would try to provide some debug logs here too, but I'm not sure how to filter with adb correctly, so only the app log will be shown.

Would be awesome if anyone could help me.

**Comment by HarelM** [comment]
Yes, I noticed that too.
Didn't bothered me much though as I didn't find my app icon that important.
I think I read somewhere that you need to set the notification to sometime like "on going task" in order to see the icon.
This is an Android behavior change that I just didn't feel like pursuing...
Feel free to open a PR if this is something you would like to push forward.

**Comment by veitbjarsch** [comment]
Okay, so this means even if I do not see the foreground message the plugin will still work as expected when the application is minimized and still records and post the location data to the app itself, right? Then it would not bother me either, but I wasn't sure about this, because I thought the notification would be mandatory to ensure that the gps location gathering will work as expected.

**Comment by HarelM** [comment]
I'm not sending data to a backend server in my app, but the location service keeps running in the background as far as I could test, yes.

**Comment by veitbjarsch** [comment]
Okay, thanks for the information. I might test it out tomorrow then. If this is true I would be more than happy, because I always found this message quite ugly.

**Comment by veitbjarsch** [comment]
Yeah after some tests I can see that's the plugin is fully functional even without the visible foreground message.
